src/content/1.7/development/webservice/tutorials/2-testing-access.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 2 - Testing access to the Webservice
+menuTitle: 2 - Testing access
+weight: 2
+---
+
+# Testing access to the Webservice
+
+## Accessing /api/
+
+Now that your access key is generated you can test your store's webservice, its endpoint is located in the `/api/` folder at the root of your installation of Prestashop.
+The quickest way to test your API is to use your browser:
+
+* If PrestaShop is installed at the root of your server, you can access the API here: http://example.com/api/
+* If PrestaShop is installed in a subfolder of your server, you can access the API here: http://example.com/prestashop/api/
+
+The shop should prompt you for a username and a password to enter. The username is the authentication key you created and **there is no password to enter**. If no permission has been set for the key, than the browser will keep asking you to enter the key indefinitely.
+
+The second and more appropriate way to access the API is to include your access key in the url, this will prevent you from entering any user name.
```

Comment:
It's not the more appropriate way to access to an API because if there is someone see the apache log or sniff browser history he can retrieve the API Key.
Using headers is a more appropriate way to access to the API :thinking:

Reply:
I agree but it's not a PR to change the behaviour the API, just a documentation about its current state ^^ Besides headers could be used if you use a proper client, but in the browser it's more complicated

Reply:
We shouldn't talk about it, and discourage this usage for security reason. We should talk about best practices only. Since it concerns an API, it should target developers, not lambda users.

Reply:
If using headers is possible with the webservice, then it should be stated. If not, then we should note that this is not recommended in production for the reasons explained by Pierre.

Reply:
Yep I found how to use with proper headers:
```
$encodedAuth = base64_encode(self::$pfAdapterUser.":".self::$pfAdapterPasswd);
curl_setopt($ch, CURLOPT_HTTPHEADER, array("Authorization: Basic ".$encodedAuth));
```
So I'll document this, and leave the less secure one only in a side note with a warning not to use for production

Reply:
Or I could remove the part where we suggest to include the key in the url altogether? If someone wants to test via the browser he can always manually enter the key in the prompt

Reply:
Leave the part from the URL in it, it's the easiest way in many use cases. Just note that it's not recommended.

Reply:
Ok thanks to you both, I divided this section into three sub part:
- Browser prompt
- Key in url (risky)
- Authorization header (recommended)

This is much better now indeed! Let me know what you think of it

Reply:
Looks good 👍